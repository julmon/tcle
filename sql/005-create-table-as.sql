SET search_path TO public, s1;
-- Test CREATE TABLE AS statement
CREATE TABLE t2 (n, l, d) USING tcleam AS SELECT n, l, d FROM t WHERE n > 6;
SELECT ctid, * FROM t2;
