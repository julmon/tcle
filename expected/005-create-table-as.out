SET search_path TO public, s1;
-- Test CREATE TABLE AS statement
CREATE TABLE t2 (n, l, d) USING tcleam AS SELECT n, l, d FROM t WHERE n > 6;
SELECT ctid, * FROM t2;
  ctid  |  n  |           l           |              d               
--------+-----+-----------------------+------------------------------
 (0,1)  | 8   | Text input number 4   | Sat Jun 27 22:56:50 2020 PDT
 (0,2)  | 10  | Text input number 5   | Sun Jun 28 22:56:50 2020 PDT
 (0,3)  | 12  | Text input number 6   | Mon Jun 29 22:56:50 2020 PDT
 (0,4)  | 14  | Text input number 7   | Tue Jun 30 22:56:50 2020 PDT
 (0,5)  | 16  | Text input number 8   | Wed Jul 01 22:56:50 2020 PDT
 (0,6)  | 18  | Text input number 9   | Thu Jul 02 22:56:50 2020 PDT
 (0,7)  | 20  | Text input number 10  | Fri Jul 03 22:56:50 2020 PDT
 (0,8)  | 22  | Text input number 11  | Sat Jul 04 22:56:50 2020 PDT
 (0,9)  | 24  | Text input number 12  | Sun Jul 05 22:56:50 2020 PDT
 (0,10) | 26  | Text input number 13  | Mon Jul 06 22:56:50 2020 PDT
 (0,11) | 28  | Text input number 14  | Tue Jul 07 22:56:50 2020 PDT
 (0,12) | 30  | Text input number 15  | Wed Jul 08 22:56:50 2020 PDT
 (0,13) | 32  | Text input number 16  | Thu Jul 09 22:56:50 2020 PDT
 (0,14) | 34  | Text input number 17  | Fri Jul 10 22:56:50 2020 PDT
 (0,15) | 36  | Text input number 18  | Sat Jul 11 22:56:50 2020 PDT
 (0,16) | 38  | Text input number 19  | Sun Jul 12 22:56:50 2020 PDT
 (0,17) | 40  | Text input number 20  | Mon Jul 13 22:56:50 2020 PDT
 (0,18) | 42  | Text input number 21  | Tue Jul 14 22:56:50 2020 PDT
 (0,19) | 44  | Text input number 22  | Wed Jul 15 22:56:50 2020 PDT
 (0,20) | 46  | Text input number 23  | Thu Jul 16 22:56:50 2020 PDT
 (0,21) | 48  | Text input number 24  | Fri Jul 17 22:56:50 2020 PDT
 (0,22) | 50  | Text input number 25  | Sat Jul 18 22:56:50 2020 PDT
 (0,23) | 52  | Text input number 26  | Sun Jul 19 22:56:50 2020 PDT
 (0,24) | 54  | Text input number 27  | Mon Jul 20 22:56:50 2020 PDT
 (0,25) | 56  | Text input number 28  | Tue Jul 21 22:56:50 2020 PDT
 (0,26) | 58  | Text input number 29  | Wed Jul 22 22:56:50 2020 PDT
 (0,27) | 60  | Text input number 30  | Thu Jul 23 22:56:50 2020 PDT
 (0,28) | 62  | Text input number 31  | Fri Jul 24 22:56:50 2020 PDT
 (0,29) | 64  | Text input number 32  | Sat Jul 25 22:56:50 2020 PDT
 (0,30) | 66  | Text input number 33  | Sun Jul 26 22:56:50 2020 PDT
 (0,31) | 68  | Text input number 34  | Mon Jul 27 22:56:50 2020 PDT
 (0,32) | 70  | Text input number 35  | Tue Jul 28 22:56:50 2020 PDT
 (0,33) | 72  | Text input number 36  | Wed Jul 29 22:56:50 2020 PDT
 (0,34) | 74  | Text input number 37  | Thu Jul 30 22:56:50 2020 PDT
 (0,35) | 76  | Text input number 38  | Fri Jul 31 22:56:50 2020 PDT
 (0,36) | 78  | Text input number 39  | Sat Aug 01 22:56:50 2020 PDT
 (0,37) | 80  | Text input number 40  | Sun Aug 02 22:56:50 2020 PDT
 (0,38) | 82  | Text input number 41  | Mon Aug 03 22:56:50 2020 PDT
 (0,39) | 84  | Text input number 42  | Tue Aug 04 22:56:50 2020 PDT
 (0,40) | 86  | Text input number 43  | Wed Aug 05 22:56:50 2020 PDT
 (0,41) | 88  | Text input number 44  | Thu Aug 06 22:56:50 2020 PDT
 (0,42) | 90  | Text input number 45  | Fri Aug 07 22:56:50 2020 PDT
 (0,43) | 92  | Text input number 46  | Sat Aug 08 22:56:50 2020 PDT
 (0,44) | 94  | Text input number 47  | Sun Aug 09 22:56:50 2020 PDT
 (0,45) | 96  | Text input number 48  | Mon Aug 10 22:56:50 2020 PDT
 (0,46) | 98  | Text input number 49  | Tue Aug 11 22:56:50 2020 PDT
 (0,47) | 100 | Text input number 50  | Wed Aug 12 22:56:50 2020 PDT
 (0,48) | 102 | Text input number 51  | Thu Aug 13 22:56:50 2020 PDT
 (0,49) | 104 | Text input number 52  | Fri Aug 14 22:56:50 2020 PDT
 (0,50) | 106 | Text input number 53  | Sat Aug 15 22:56:50 2020 PDT
 (0,51) | 108 | Text input number 54  | Sun Aug 16 22:56:50 2020 PDT
 (0,52) | 110 | Text input number 55  | Mon Aug 17 22:56:50 2020 PDT
 (1,1)  | 112 | Text input number 56  | Tue Aug 18 22:56:50 2020 PDT
 (1,2)  | 114 | Text input number 57  | Wed Aug 19 22:56:50 2020 PDT
 (1,3)  | 116 | Text input number 58  | Thu Aug 20 22:56:50 2020 PDT
 (1,4)  | 118 | Text input number 59  | Fri Aug 21 22:56:50 2020 PDT
 (1,5)  | 120 | Text input number 60  | Sat Aug 22 22:56:50 2020 PDT
 (1,6)  | 122 | Text input number 61  | Sun Aug 23 22:56:50 2020 PDT
 (1,7)  | 124 | Text input number 62  | Mon Aug 24 22:56:50 2020 PDT
 (1,8)  | 126 | Text input number 63  | Tue Aug 25 22:56:50 2020 PDT
 (1,9)  | 128 | Text input number 64  | Wed Aug 26 22:56:50 2020 PDT
 (1,10) | 130 | Text input number 65  | Thu Aug 27 22:56:50 2020 PDT
 (1,11) | 132 | Text input number 66  | Fri Aug 28 22:56:50 2020 PDT
 (1,12) | 134 | Text input number 67  | Sat Aug 29 22:56:50 2020 PDT
 (1,13) | 136 | Text input number 68  | Sun Aug 30 22:56:50 2020 PDT
 (1,14) | 138 | Text input number 69  | Mon Aug 31 22:56:50 2020 PDT
 (1,15) | 140 | Text input number 70  | Tue Sep 01 22:56:50 2020 PDT
 (1,16) | 142 | Text input number 71  | Wed Sep 02 22:56:50 2020 PDT
 (1,17) | 144 | Text input number 72  | Thu Sep 03 22:56:50 2020 PDT
 (1,18) | 146 | Text input number 73  | Fri Sep 04 22:56:50 2020 PDT
 (1,19) | 148 | Text input number 74  | Sat Sep 05 22:56:50 2020 PDT
 (1,20) | 150 | Text input number 75  | Sun Sep 06 22:56:50 2020 PDT
 (1,21) | 152 | Text input number 76  | Mon Sep 07 22:56:50 2020 PDT
 (1,22) | 154 | Text input number 77  | Tue Sep 08 22:56:50 2020 PDT
 (1,23) | 156 | Text input number 78  | Wed Sep 09 22:56:50 2020 PDT
 (1,24) | 158 | Text input number 79  | Thu Sep 10 22:56:50 2020 PDT
 (1,25) | 160 | Text input number 80  | Fri Sep 11 22:56:50 2020 PDT
 (1,26) | 162 | Text input number 81  | Sat Sep 12 22:56:50 2020 PDT
 (1,27) | 164 | Text input number 82  | Sun Sep 13 22:56:50 2020 PDT
 (1,28) | 166 | Text input number 83  | Mon Sep 14 22:56:50 2020 PDT
 (1,29) | 168 | Text input number 84  | Tue Sep 15 22:56:50 2020 PDT
 (1,30) | 170 | Text input number 85  | Wed Sep 16 22:56:50 2020 PDT
 (1,31) | 172 | Text input number 86  | Thu Sep 17 22:56:50 2020 PDT
 (1,32) | 174 | Text input number 87  | Fri Sep 18 22:56:50 2020 PDT
 (1,33) | 176 | Text input number 88  | Sat Sep 19 22:56:50 2020 PDT
 (1,34) | 178 | Text input number 89  | Sun Sep 20 22:56:50 2020 PDT
 (1,35) | 180 | Text input number 90  | Mon Sep 21 22:56:50 2020 PDT
 (1,36) | 182 | Text input number 91  | Tue Sep 22 22:56:50 2020 PDT
 (1,37) | 184 | Text input number 92  | Wed Sep 23 22:56:50 2020 PDT
 (1,38) | 186 | Text input number 93  | Thu Sep 24 22:56:50 2020 PDT
 (1,39) | 188 | Text input number 94  | Fri Sep 25 22:56:50 2020 PDT
 (1,40) | 190 | Text input number 95  | Sat Sep 26 22:56:50 2020 PDT
 (1,41) | 192 | Text input number 96  | Sun Sep 27 22:56:50 2020 PDT
 (1,42) | 194 | Text input number 97  | Mon Sep 28 22:56:50 2020 PDT
 (1,43) | 196 | Text input number 98  | Tue Sep 29 22:56:50 2020 PDT
 (1,44) | 198 | Text input number 99  | Wed Sep 30 22:56:50 2020 PDT
 (1,45) | 200 | Text input number 100 | Thu Oct 01 22:56:50 2020 PDT
(97 rows)

